config/settings/prod.py

```
@@ -1,14 +1,47 @@
+
+DEBUG = False
 ALLOWED_HOSTS = ["*"]
 
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
-    }
+# 환경변수에서 데이터베이스 설정 가져오기
+DB_TYPE = os.getenv('DB_TYPE', 'mysql')
```

Ruff (0.12.2)

`os` may be undefined, or defined from star imports

(F405)